utils/hash_algo_djb2.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2018-2019 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+#ifndef HASH_ALGO_DJB2_H_
+#define HASH_ALGO_DJB2_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+static inline uint32_t hash_algo_djb2(const char const* str) {
```

Comment:
Since djb2 hash function is not originated from TA, we shall mention its upstream work. Add the following hyperlink: http://www.cse.yorku.ca/~oz/hash.html